### ISSUE TITLE
Threaded rendering adds a frame of input lag

### ISSUE THREAD

**Issue by Tatsuya79** [libretro/flycast#738]
It takes 1 more frame to see a reaction on screen when threaded rendering is activated (using the "frame step" K hotkey to count frames).

Tested with the vulkan renderer, Border Down Naomi, win7 x64 latest RA and flycast nightly.

**Comment by flyinghead** [comment]
This doesn't happen with the GL renderer?

**Comment by Tatsuya79** [comment]
Yes, same behaviour, 4 frames to react with glcore and threaded off, 5 with threaded on.

**Comment by RaduNastase** [comment]
Are we talking Core settings or Retroarch settings?

**Comment by Tatsuya79** [comment]
core option

**Comment by Immersion95** [comment]
Yes I can confirm. Why is it ?

**Comment by Immersion95** [comment]
Tested with latest builds and the problem is still there on all backend (GL + Vulkan + DX11).
